Integrations/integration-Symantec_Site_Review.yml

```diff
@@ -0,0 +1,205 @@
+        sites = demisto.args().get('urls')
+        site_details = []
+        if isinstance(sites, basestring):
+            sites = sites.split(',')
+        for site in sites:
+            raw_lookup = site_lookup(site)
+            site_detail = {}
+            md_headers = ['URL','CategorizationID','CategorizationName']
+            site_detail['URL'] = raw_lookup['url']
+            site_detail['CategorizationID'] = raw_lookup['categorization'][0]['num']
+            site_detail['CategorizationName'] = raw_lookup['categorization'][0]['name']
+            if raw_lookup['resolvedDetail']['ipAddress']:
+                site_detail['ResolvedIP'] = raw_lookup['resolvedDetail']['ipAddress']
+                md_headers.append('ResolvedIP')
+            if raw_lookup['threatriskLevel']:
+                site_detail['ThreatRiskLevel'] = raw_lookup['threatriskLevel']
+            else:
+                site_detail['ThreatRiskLevel'] = ""
+            md_headers.append('ThreatRiskLevel')
+            site_details.append(site_detail)
+
+        md = tableToMarkdown('Lookup Categorization', site_details, md_headers)
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': site_details,
+            'HumanReadable': md,
+            'EntryContext': {
+                'Symantec.Site': site_details
+            }
+        })
+
+    def test():
+
+        res = http_request('GET', BASE_URL)
+        demisto.results('ok')
+
+    """
+    CODE EXECUTION STARTS HERE
+
+    demisto.command() returns the name of the command which executed now
+    """
+    LOG('command is %s' % (demisto.command(), ))
+    try:
+        if demisto.command() == 'test-module':
+            """
+            demisto.command() will return 'test-module' when the Test button in integration page clicked
+            """
+            test()
+        elif demisto.command() == 'symantec-site-lookup':
+            site_lookup_command()
+        elif demisto.command() == 'symantec-site-list-risklevels':
+            list_risk_levels_command()
+
+    except Exception, e:
+        LOG(e.message)
+        LOG.print_log()
+        raise
+  type: python
+  commands:
+  - name: symantec-site-lookup
+    arguments:
+    - name: urls
+      required: true
+      description: URLs to lookup. Can be a single value or comma separated  list
+        of values.
+    outputs:
+    - contextPath: Symantec.Site.CategorizationID
+      description: Site Categorization Number
+      type: string
+    - contextPath: Symantec.Site.CategorizationName
+      description: Site Categorization Name
+      type: string
+    - contextPath: Symantec.Site.ResolvedIP
+      description: Site Resolved IP Address
+      type: string
+    - contextPath: Symantec.Site.ThreatRiskLevel
+      description: Site Threat Risk Level
+      type: string
+    description: Symantec WebPulse Site Review Request
+  - name: symantec-site-list-risklevels
```

Comment:
I'm not sure what this function does, could you explain?

Reply:
Gives a list of Risk Levels and their description.  helps users understand the Risk Level value.